README.md

```diff
@@ -55,9 +55,46 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 ### Playbook
 
 ```yaml
+---
 - hosts: all
+  any_errors_fatal: true
   roles:
-    - cloudalchemy.alertmanager
+    - ansible-alertmanager
+  vars:
+    alertmanager_binary_local_dir: '/tmp/alertmanager-linux-amd64'
+    alertmanager_config_dir: /opt/am/etc
+    alertmanager_db_dir: /opt/am/lib
+    alertmanager_web_listen_address: '127.0.0.1:9093'
+    alertmanager_web_external_url: 'http://localhost:9093/alertmanager'
+    alertmanager_resolve_timeout: 10m
+    alertmanager_slack_api_url: "http://example.com"
```

Comment:
In the provided example, this option is not used as slack receiver has its own `api_url` option set. It might be good to add another receiver which doesn't use `api_url` and route anything there.